3.Tests/UI/Features/Item/Create.feature

```diff
@@ -5,15 +5,15 @@ Feature: Create an Item in a Project
         Given the user is logged in
     #And the user has an existing project
 
-    @create.project.NewProject
+    @create.project.NewProject @Vale
```

Comment:
What the @Vale tag is? Should be removed right?

Reply:
It was to do isolated test to just this one. But now is deleted. Thank you.